scripts/ashell-monkey.py

```diff
@@ -0,0 +1,99 @@
+# This tool can test many JS test files at once using ashell. To use, flash a
+# ZJS device with ashell and start it. Then run this script, passing in the
+# files you want to test, as well as the serial port the board is connected to:
+
+#    python scripts/ashell-monkey.py <port> [print] <list of files>
+
+# The port parameter should be something like "/dev/ttyACM0"
+# After the port, you can pass in an optional "print" string, which will print
+# all the ashell output.
+# After the port and print parameters pass in a list of files you would like
+# to test. These should be ZJS test files which end in printing:
+
+#    "TOTAL: X of Y tests passed"
+
+# This is the string that ashell-monkey looks for to determine if the test is
+# over, as well as if it passed or not. Here is an example of how you might
+# run this script:
+
+#    python scripts/ashell-monkey /dev/ttyACM0 print tests/test-buffer.js \
+#        tests/test-fs.js tests/test-error.js tests/test-event.js
+
+# This would run the buffer, fs, error, and event test scripts found under
+# tests/. At the end you should get a message if any tests failed or if all
+# succeeded.
+
+import serial
```

Comment:
Add documentation above that you need to "pip3 install pyserial" for this to work.